packages/panna-sdk/src/react/components/send/send-processing-step.tsx

```diff
@@ -41,7 +41,7 @@ export function SendProcessingStep({ form }: SendProcessingStepProps) {
         client,
         chain,
         to: form.getValues('recipientAddress') as Address,
-        value: BigInt(toWei(cryptoAmount))
+        value: BigInt(toWei(cryptoAmount) * 100000000000000000n) // Convert ETH to wei
```

Comment:
`toWei` already converts ETH to wei (i.e. multiplies with 10 ** 18) and returns a BigInt.
In addition, the `100000000000000000n` is one zero short (has 17 zeros). It's always good to use something more readable and less error prone like: `BigInt(10 ** 18)` or `BigInt(Math.pow(10, 18))` instead.

```suggestion
        value: toWei(cryptoAmount) // Convert ETH to wei
```